src/spatialdata_plot/pl/basic.py

```diff
@@ -589,57 +557,79 @@ def show(
                     label_transformation = list(label_transformation.values())[0]
                     sdata.labels[key] = transform(sdata.labels[key], label_transformation)
 
-        # get biggest image after transformations to set ax size
-        x_dims = []
-        y_dims = []
+        # Get extent of data to be plotted
+        want_images_extent: bool = False
+        want_labels_extent: bool = False
+        want_points_extent: bool = False
+        want_shapes_extent: bool = False
```

Comment:
think this should be a mapping, namedtuple, or dataclass

Reply:
Mapping would add more lines since it's a one-off thing, tuple would be immutable and dataclass also a little overkill for a one-off thing. Refactored to this instead

<img width="514" alt="image" src="https://user-images.githubusercontent.com/12818242/232467843-5e218533-9aa5-45fe-97b7-2dc62f4e6e3b.png">